app/bundles/CampaignBundle/Tests/CampaignTestAbstract.php

```diff
@@ -80,9 +76,12 @@ protected function initCampaignModel()
         $eventCollector    = $this->createMock(EventCollector::class);
         $membershipBuilder = $this->createMock(MembershipBuilder::class);
 
-        $campaignModel = new CampaignModel($leadModel, $leadListModel, $formModel, $eventCollector, $membershipBuilder);
+        $contactTracker = $this->getMockBuilder(ContactTracker::class)
```

Comment:
Let's use the shorter syntax for the newly added code if possible. It improves readability.
```suggestion
        $contactTracker = $this->createMock(ContactTracker::class);
```